docs/user-guide/logging.md

```diff
@@ -6,35 +6,35 @@ Logging in HELICS is normally handled through an independent thread. The thread
+- `HELICS_PROPERTY_INT_LOG_LEVEL` General logging level applicable to both file and console logs
 
-- `helics_property_int_file_log_level` Level to log to the file
+- `HELICS_PROPERTY_INT_file_log_level` Level to log to the file
 
-- `helics_property_int_console_log_level` Level to log to the console
+- `HELICS_PROPERTY_INT_console_log_level` Level to log to the console
 
 These properties can be set using the API interface functions
 
 ```c
-helicsFederateInfoSetIntegerProperty(fi,helics_property_int_log_level, helics_log_level_data,&err);
+helicsFederateInfoSetIntegerProperty(fi,HELICS_PROPERTY_INT_LOG_LEVEL, HELICS_LOG_LEVEL_data,&err);
 ```
 
 ```python
-h.helicsFederateInfoSetIntegerProperty(fi,h.helics_property_int_log_level, h.helics_log_level_data)
+h.helicsFederateInfoSetIntegerProperty(fi,h.HELICS_PROPERTY_INT_LOG_LEVEL, h.HELICS_LOG_LEVEL_data)
 ```
 
 NOTE: logging level properties set in a federateInfo will apply to a core as well if it is the first federate registered in the core. After registration log level properties must be set separately for the core and federate.
 
 There are several levels used inside HELICS for logging
 
-- `helics_log_level_no_print` Don't print anything
-- `helics_log_level_error` Error and faults from within HELICS
-- `helics_log_level_warning` Warning messages of things that might be incorrect or unusual
-- `helics_log_level_summary` Summary messages on startup and shutdown. The Broker will also generate a summary with the number of federates connected and a few other items of information
-- `helics_log_level_connections` Log a message for each connection event (federate connection/disconnection)
-- `helics_log_level_interfaces` Log messages when interfaces, such as endpoints, publications, and filters are created
-- `helics_log_level_timing` Log messages related to timing information such as mode transition and time advancement
-- `helics_log_level_data` Log messages related to data passage and information being sent or received
-- `helics_log_level_trace` Log all internal message being sent
+- `HELICS_LOG_LEVEL_no_print` Don't print anything
+- `HELICS_LOG_LEVEL_ERROR` Error and faults from within HELICS
+- `HELICS_LOG_LEVEL_warning` Warning messages of things that might be incorrect or unusual
+- `HELICS_LOG_LEVEL_summary` Summary messages on startup and shutdown. The Broker will also generate a summary with the number of federates connected and a few other items of information
+- `HELICS_LOG_LEVEL_connections` Log a message for each connection event (federate connection/disconnection)
+- `HELICS_LOG_LEVEL_interfaces` Log messages when interfaces, such as endpoints, publications, and filters are created
+- `HELICS_LOG_LEVEL_timing` Log messages related to timing information such as mode transition and time advancement
+- `HELICS_LOG_LEVEL_data` Log messages related to data passage and information being sent or received
+- `HELICS_LOG_LEVEL_trace` Log all internal message being sent
```

Comment:
Why are these not all just uppercase, e.g. `HELICS_LOG_LEVEL_NO_PRINT`

Reply:
probably because the automated fix didn't catch the names in quotes quite as well,  I will fix that.